internal/internal.go

```diff
@@ -38,3 +40,11 @@ func TimestampToTime(ts *timestamp.Timestamp) (t time.Time) {
 	}
 	return time.Unix(ts.Seconds, int64(ts.Nanos))
 }
+
+func TimestampToUnixnano(ts *timestamp.Timestamp) data.TimestampUnixNano {
+	return data.TimestampUnixNano(uint64(TimestampToTime(ts).UnixNano()))
+}
+
+func UnixnanoToTimestamp(u uint64) *timestamp.Timestamp {
```

Comment:
Would it make sense for this to accept data.TimestampUnixNano instead of uint64 for symmetry?